src/Drivers/Saman/Saman.php

```diff
@@ -52,13 +52,18 @@ public function purchase()
             'MID' => $this->settings->merchantId,
             'ResNum' => $this->invoice->getUuid(),
             'Amount' => $this->invoice->getAmount() * 10, // convert to rial
+            'AdditionalData1' => $this->invoice->getDetails()[0], // used for getting reports, due to https://www.sep.ir/Data/Sites/1/media/token98.pdf
```

Comment:
we cant use `getDetails()[0]` in all drivers! it can cause other drivers break!
I thinks its better to be like the below:

```php
$this->invoice->getDetails()['AdditionalData1'] ?? getDetails()['additional_data1'];
```